build.yaml

```diff
@@ -1584,6 +1584,25 @@ steps:
            --verbose \
            site/www/* \
            /io/www
+
+     # Grab the latest release of the docs
+     LATEST_02_SHA=$(gsutil cat gs://hail-common/builds/0.2/latest-hash/cloudtools-5-spark-2.4.0.txt 2>/dev/null)
+     if [[ -z "$LATEST_02_SHA" ]]; then
+       echo "Failed to retrieve latest sha for hail docs"
+       exit 1
+     fi
+     gsutil cat gs://hail-common/builds/0.2/docs/hail-0.2-docs-${LATEST_02_SHA}.tar.gz |
+         tar zxvf - -C /io/www --strip-components=1
```

Comment:
This will fail on the commit that creates a new hail PyPI version. This file is updated by `deploy`, which runs long after the site is deployed. The docs are uploaded by `deploy_hail` (confusing name, my bad).

Hmm. Maybe just make the site deployment depend on `deploy`. I think that makes everything work. latest-hash will be up to date. The docs will be published.

Reply:
Does `deploy` actually deploy to PyPi? Wouldn't we not want to do this every time we push out a change to the site? Or do you mean depend on `deploy_hail`?

Reply:
`deploy` pushes to PyPI, if there is a new PyPI version. Otherwise it does nothing.